scripts/test-instance/run-test

```diff
@@ -37,3 +37,4 @@ wait $PID # Wait until child process is exited
 trap - TERM INT
 
 wait $PID # In case an interrupt signal is received
+EOF
```

Comment:
This will cause a failure. Line 10 is already closing the here document. We can block comment lines between 4 and 8 instead of a here document.